packages/augur-ui/src/modules/reporting/common.styles.less

```diff
@@ -599,6 +604,11 @@ div.ReportingSubheaders {
   max-width: 482px;
   padding: @size-32 @size-24;
 
+  @media @breakpoint-mobile-new {
```

Comment:
already mentioned, don't like the `@breakpoint-mobile-new` naming isn't descriptive enough